spec/api/v1/merchants/find_controller_spec.rb

```diff
@@ -0,0 +1,107 @@
+require "rails_helper"
+
+RSpec.describe "merchants find controller" do
+  fixtures :merchants
+  context "finds one merchant record" do
+
+    scenario "by id" do
+      merchant = merchants(:one)
+      get "/api/v1/merchants/find?id=#{merchant.id}"
+
+      assert_response :success
+
+      response_merchant = JSON.parse(response.body)
```

Comment:
* look into ways of refactoring lines that are repeated
*  maybe a before action in rspec?